MIT

```diff
@@ -1,9 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c), 2015, ECOLE POLYTECHNIQUE FEDERALE DE LAUSANNE
-(Theory and Simulation of Materials (THEOS) and National Centre for
-Computational Design and Discovery of Novel Materials (NCCR MARVEL)),
-Switzerland and ROBERT BOSCH LLC, USA. All rights reserved.
+Copyright 2020
```

Comment:
Shouldn't this state how the copyright belongs to? And if so, who does it belong to?

Reply:
It does normally, but it would be very difficult to figure out in any way that actually has any meaning.  By default the writer of each line of code owns the copyright to that line - that fact might be modified by their contract with an employer or some other contract so I figured I couldn't really put anything in there that I hadn't just pulled out of my arse.  In any case, if there was a legal issue what is written there wouldn't override whatever laws normally govern it.